docs/repositories.md

```diff
@@ -0,0 +1,71 @@
+# Generated Bazel repositories
+
+rules_nodejs produces several repositories for you to reference.
+Bazel represents your Workspace as one repository, and code fetched or installed from outside your workspace lives in other repositories.
+These are referenced with the `@repo` syntax in your BUILD files.
+
+## @nodejs
+
+This repository is created by calling the `node_repositories` function. It contains the node, npm, and yarn programs.
+
+As always, `bazel query` is useful for learning about what targets are available.
+
+```sh
+$ bazel query @nodejs//...
+@nodejs//:node
+...
+```
+
+You don't typically need to reference the `@nodejs` repository from your BUILD files because it's used behind the scenes
+to run node and fetch dependencies.
+
+Some ways you can use this:
+
+- Run the Bazel-managed version of node: `bazel run @nodejs//:node path/to/program.js`
+- Run the Bazel-managed version of npm: `bazel run @nodejs//:npm`
+- Run the Bazel-managed version of yarn: `bazel run @nodejs//:yarn`
+- Install dependencies from nested package.json file(s) which were passed to `node_repositories#package.json`
+  - using npm: `bazel run @nodejs//:npm_node_repositories install`
+  - using yarn: `bazel run @nodejs//:yarn_node_repositories`
+
+## @npm
+
+This repository is created by calling the `npm_install` or `yarn_install` function.
```

Comment:
nit: function in your WORKSPACE file. First time users will not know where to look.